content/en/tracing/llm_observability/api.md

```diff
@@ -0,0 +1,331 @@
+|---------|-----------------------------|------------------------------------------|------------|
+| ID      | string                      | Response UUID generated upon submission. | Yes        |
+| metrics | [[EvalMetric](#evalmetric)] | A list of eval metrics.                  | Yes        |
+{{% /tab %}}
+
+{{% tab "Example" %}}
+{{< code-block lang="json" >}}
+{
+  "data": {
+    "type": "evaluation_metric",
+    "id": "456f4567-e89b-12d3-a456-426655440000",
+    "attributes": {
+      "metrics": [
+        {
+          "id": "d4f36434-f0cd-47fc-884d-6996cee26da4",
+          "span_id": "61399242116139924211",
+          "trace_id": "13932955089405749200",
+          "timestamp": 1609459200,
+          "metric_type": "categorical",
+          "label": "Sentiment",
+          "categorical_value": "Positive"
+        },
+        {
+          "id": "cdfc4fc7-e2f6-4149-9c35-edc4bbf7b525",
+          "span_id": "20245611112024561111",
+          "trace_id": "13932955089405749200",
+          "metric_type": "score",
+          "label": "Accuracy",
+          "score_value": 3
+        }
+      ]
+    }
+  }
+}
+{{< /code-block >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+### API standards
+
+#### Attribute
+
+| Field   | Type         | Description                                         |
+|---------|--------------|-----------------------------------------------------|
+| metrics [*required*] | [[EvalMetric](#evalmetric)] | A list of eval metrics for a given prompt-response pair. |
+
+#### EvalMetric
+
+| Field                  | Type   | Description  |
+|------------------------|--------|--------------|
+| ID                     | string | Evaluation metric UUID (generated upon submission). |
+| span_id [*required*]    | string | The ID of the span that this eval metric is associated with. This should be the span ID of the root span. |
+| trace_id [*required*]   | string | The ID of the trace that this eval metric is associated with. |
+| timestamp              | int64  | A UTC UNIX timestamp representing the time the request was sent. |
+| metric_type [*required*]| string | The type of evaluation metric: `"categorical"` or `"score"`. |
+| label [*required*]      | string | The unique name or label for the provided evaluation metric. |
+| categorical_value [*required if the metric_type is "score"*]    | string | A string representing the category that the evaluation metric belongs to. |
+| score_value [*required if the metric_type is "score"*]    | number | A score value of the evaluation metric. |
+| flagged                | boolean| Flag content as inappropriate or incorrect. |
+| annotation             | string | A generic string note about the provided evaluation metric. |
+
+#### EvalMetricsRequestData
+
+| Field      | Type            | Description  | 
+|------------|-----------------|--------------|
+| type [*required*]      | "evaluation_metric" | Identifier for the request. Value depends on the API. |
```

Comment:
The type "evaluation_metric" seems strange, is there something else that should go here? "Value depends on the API" is also confusing, can you clarify?

Reply:
So if someone is using the evaluation metrics endpoint they have to specify the `type` as `evaluation_metric`. It could be more clear if we directly tell them it has to be exactly `evaluation_metric`?

For "Value Depends on the API" -- agree it's confusing. It's supposed to communicate that the `type` depends on what endpoint you are using, e.g. the `POST /spans` the value would be `span`.

here's what we have for the `POST /spans` endpoint, should we also change this to be something similar?
<img width="713" alt="image" src="https://github.com/DataDog/documentation/assets/42917263/4453db83-9f2c-4eb4-b9b7-7ab0b32033be">